index.js

```diff
@@ -16,6 +19,7 @@ function get_new_question() {
         let current_movie = remaining_movies.pop()
         let quote_element = document.getElementById('quote')
         quote_element.textContent = current_movie.quote
+        console.log(current_movie.quote)
```

Comment:
No console logs should make it on to the codebase. Only for bug fixing

Reply:
done!